site/tutorials/tutorial-six-dotnet.md

```diff
@@ -100,14 +98,14 @@ channel.BasicPublish(exchange: "",
 > a message. Most of the properties are rarely used, with the exception of
 > the following:
 >
-> * `deliveryMode`: Marks a message as persistent (with a value of `2`)
->    or transient (any other value). You may remember this property
->    from [the second tutorial](tutorial-two-dotnet.html).
-> * `contentType`: Used to describe the mime-type of the encoding.
+> * `DeliveryMode`: Marks a message as persistent (with a value of `2`)
+>    or transient (any other value). This is similar to the `Persistent` 
```

Comment:
It's not "similar". `Persistent` is a more beginner-friendly way of setting `DeliveryMode`.

Reply:
Thanks for taking a look. How about "This is a more general form of the `Persistent` property from [the second tutorial]."?
Then users who want to know more could always look it up.